### ISSUE TITLE
Accessibility - Screen requires horizontal scrolling 

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#18]
### Android 2.2.11

**Describe the bug**
The screen, particularly in Events, requires horizontal scrolling. This can interfere with the swiping actions used by screen reader users. Horizontal scrolling should be avoided. 

**To Reproduce**
1. Go to Events 
2. Scroll vertically to just below Academic Evening Class
3. View the available times (requires horizontal scrolling)

**Expected behavior**
Screen elements should be stacked so that  vertical scrolling is the only scrolling action required. 

**Screenshots**
See below

**Desktop (please complete the following information):**
 - OS: Android
 - Version  2.2.11

**Smartphone (please complete the following information):**
 - Device: Google Pixel 3
 - OS: Android
 - Version  2.2.11

<img width="475" alt="HorizontalScrolling2-mrkup" src="https://user-images.githubusercontent.com/12497650/91500433-252b9c00-e889-11ea-87fb-fffd3df1a132.png">

**Comment by TodorBachvarov** [comment]
@timo54 @pmarkhennessy We cannot do this. If we disable the horizontal scrolling, the widgets won't fit within the screen (even if we size down the texts and the paddings, if the user uses Accessibility large text the widgets won't fit again). If we want to remove the  horizontal scrolling so to have only vertical scrolling - probably we will want to do the same for all panels which have vertical + horizontal scrolling. We will need proper redesign of these panels, so probably this should be reassigned and discussed with the design team.

**Comment by acovher2** [comment]
@TodorBachvarov Hello I'm the new tester on the IL App.  Where does this redesign stand?

**Comment by acovher2** [comment]
This is verified as not occurring in 2.6.28, yet the rest of the page becomes invisible with large fonts so there will be a new issue for that.
